### DISCUSSION TITLE
Effective deletion of product images from filesystem

### DISCUSSION THREAD

**Discussion by empiricompany** [OpenMage/magento-lts#3967] [Ideas]
@fballiano What do you think about including into the core this feature you implemented here?
https://github.com/fballiano/openmage-autodelete-product-images/

I think it's crucial and an old 'bug' we've been carrying around for years. There's no reason to not delete them directly.
A more complex task is deleting all cached generated images, but you should have already addressed it here:
https://github.com/fballiano/openmage-image-cleaner/commit/ab54703227e9900439bc6a96e7106ea1f738c9f1

**Comment by sreichel** [comment]
Everything we put into core has to be maintained.

We can add it to composer suggestion. (PR welcome)

And its mentioned here: https://docs.openmage.org/users/modules/images/#fballianoopenmage-image-cleaner
